scripts/talos/README.md

```diff
@@ -1,12 +1,24 @@
-# Talos
+# Talos - package bumping bot
+
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Workinnng.md)
```

Comment:
Typo. `https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md`

Reply:
@mykeels done